src/NServiceBus.Testing/Handler.cs

```diff
@@ -105,16 +108,7 @@ public Handler<T> ExpectNotSendLocal<TMessage>(Func<TMessage, bool> check)
         /// </summary>
         public Handler<T> ExpectReturn<TEnum>(Func<TEnum, bool> check)
         {
-            expectedInvocations.Add(new ExpectedReturnInvocation<TEnum> { Check = check });
-            return this;
-        }
-
-        /// <summary>
-        /// Check that the handler sends the given message type to the appropriate destination.
-        /// </summary>
-        public Handler<T> ExpectSendToDestination<TMessage>(Func<TMessage, Address, bool> check)
-        {
-            expectedInvocations.Add(new ExpectedSendToDestinationInvocation<TMessage> { Check = check });
+            //testableMessageHandlerContext.ExpectedInvocations.Add(new ExpectedReturnInvocation<TEnum> { Check = check });
```

Comment:
this should be obsoleted since SendToDestination belongs to the Gateway package.